### ISSUE TITLE
TUD: Use vets-gov-write Team Membership Permit Login

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#37734]
# Description
In order to mimic the permissions required for access to va.gov-team-sensitive, we need to base login permission on membership in [vets.gov-write](https://github.com/orgs/department-of-veterans-affairs/teams/vets-gov-write).

[Useful documentation](https://docs.github.com/en/rest/reference/teams#get-team-membership-for-a-user)???

## Acceptance Criteria
- [x] User who are members of [vets.gov-write](https://github.com/orgs/department-of-veterans-affairs/teams/vets-gov-write) authenticate with TUD successfully
- [x] Users who are a member of DSVA org but not vets-gov-write are not authenticated

**Comment by holdenhinkle** [comment]
PR - https://github.com/department-of-veterans-affairs/vsp-test-user-dashboard/pull/130

It seems the `vets-gov-write` team is not visible to users who are a part of it. I submitted a request to have that team be made visible - https://github.com/department-of-veterans-affairs/github-user-requests/issues/9962

This ticket is on hold until that team is made visible.

**Comment by holdenhinkle** [comment]
I added a comment to the issue, just checking in with them - https://github.com/department-of-veterans-affairs/github-user-requests/issues/9962#issuecomment-1057386217

**Comment by holdenhinkle** [comment]
I was given the wrong team name.

Karl Brown brought that to my attention in this thread - https://dsva.slack.com/archives/CU1E4CX9U/p1646258108339809

I'm now authorizing a user if their state for the team is `active`.

This is merged.

**Comment by holdenhinkle** [comment]
It seems that a number of people are not members of this team, including Ryan Beckwidth and a few other users we onboarded to TUD recently, so they can't access TUD at this time.

**Comment by holdenhinkle** [comment]
I updated authorization so you must be a collaborator on the `va.gov-team-sensitive`  repo.

I also added a little update about TUD OAuth to [this confluence page](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1910702416/Using+the+Test+User+Dashboard) which will eventually be published [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/Using-the-Test-User-Dashboard.1910702416.html) when someone pushes the recent changes.

It doesn’t make sense to add info about OAuth to the Help page in TUD because it’s locked down (they have to be authorized to see it).
